### ISSUE TITLE
How to normalize epsilon when I  must use the normalization for Imagnet pretrained model?

### ISSUE THREAD

**Issue by Stephen-Hao** [Harry24k/adversarial-attacks-pytorch#72]
when I use Imagenet pretrained resnet50 ,I must normalize the input for mean=[0.485, 0.456, 0.406],  std=[0.229, 0.224, 0.225]. How to normalize epsilon for the condition ? Thanks a lot.

**Comment by Harry24k** [comment]
Please refer to https://github.com/Harry24k/adversarial-attacks-pytorch/issues/69
